src/scene/Scene.ts

```diff
@@ -11,13 +11,53 @@ import {
 } from "../element";
 import { LinearElementEditor } from "../element/linearElementEditor";
 import { isFrameElement } from "../element/typeChecks";
+import { getSelectedElements } from "./selection";
+import { AppState } from "../types";
+import { Assert, SameType } from "../utility-types";
 
 type ElementIdKey = InstanceType<typeof LinearElementEditor>["elementId"];
 type ElementKey = ExcalidrawElement | ElementIdKey;
 
 type SceneStateCallback = () => void;
 type SceneStateCallbackRemover = () => void;
 
+type SelectionHash = string & { __brand: "selectionHash" };
+
+type getSelectedElementsOpts = {
```

Comment:
Is it necessary to redefine the type and not use something like `Parameters<typeof getSelectedElements>`?

Reply:
we can (`Parameters<InstanceType<typeof Scene>["getSelectedElements"]>[0]`)